src/components/customerService/noticeDetail/NoticeDetailBundle.tsx

```
@@ -0,0 +1,40 @@
+import { useGetNoticeDetail } from '../../../hooks/useGetNoticeDetail';
+import * as S from './NoticeDetailBundle.styled';
+import NoticeDetailBottom from './bottom/NoticeDetailBottom';
+import NoticeDetailContent from './content/NoticeDetailContent';
+import NoticeDetailHeader from './header/NoticeDetailHeader';
+import Spinner from '../../mypage/Spinner';
+
+export default function NoticeDetailBundle() {
+  const location = useLocation();
+  const { noticeId } = useParams();
+
+  const id = noticeId || String(location.state && location.state.id);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)